### DISCUSSION TITLE
OSV debian affected or not

### DISCUSSION THREAD

**Discussion by dioxygene** [DependencyTrack/dependency-track#4099] [Q&A]
hi
i have the version 4.11.0 of DT.

i have a component :
![image](https://github.com/user-attachments/assets/92652cb9-6d25-40ef-a90c-4e3836480b3d)

and this vulnerability is raised : 

![image](https://github.com/user-attachments/assets/968f70d7-719a-439a-8c6e-14a26e6066da)

the affected component of the vulnerability
![image](https://github.com/user-attachments/assets/3e33afc7-50e9-46cc-90b8-8614c6f5c36a)

the debian bulletin https://security-tracker.debian.org/tracker/DSA-5123-1

as we can see the version 5.2.4-1+deb10u1 fixe the issue.

could you tell me if it's an OSV issue or a DT issue ?

thanks

**Comment by nscuro** [comment]
This is likely a bug in DT. Without having tested this particular constellation, I think the `<5.2.5-2.1~deb11u1` is what's getting matched here.

DT's current version comparison is generic and unable to detect the various ins and outs of ecosystem-specific versioning. #2826 would help to address that.

**Comment by dioxygene** [comment]
Thanks for your answers
